src/ai/susi/server/api/cms/ListSkillService.java

```diff
@@ -48,6 +49,22 @@ public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Aut
         JSONArray jsonArray = new JSONArray();
         JSONObject json = new JSONObject(true);
         JSONObject skillObject = new JSONObject();
+        String countString = call.get("count", null);
```

Comment:
You get get count here directly. call.get() returns the the data type as per the default value.